src/pages/changelog/index.md

```diff
@@ -36,7 +39,7 @@ Plug-Ins menu lists the installed UXP plugins and their panel entrypoints.
     - set()
     - values()
 - <b>(Photoshop Only)</b> GUID for uniquely identifying a Creative Cloud User.
-- Adobe [Extensibility Metadata Platform (XMP)](/indesign/uxp/reference/uxp-api/reference-js/Modules/uxp/XMP/getting-started/) support
+- <b>(Photoshop Only)</b> Adobe Extensibility Metadata Platform (XMP) support.
```

Comment:
This change has been done on Padma's request to update the XMP support for InDesign.